lib/business_logic/cats_fact_bloc/content_event.dart

```diff
@@ -0,0 +1,5 @@
+part of 'content_bloc.dart';
+
+abstract class ContentEvent {}
+
+class LoadingContentEvent extends ContentEvent {}
```

Comment:
make a naming of event a verb, f.e, LoadContentEvent. Also would be nice to rename, content prefix is not describive. I would suggest Home or Main. And keep naming all over classes - HomeScreen, HomeLayout, HomeBloc, HomeEvent etc